patches/react-native-reanimated+2.9.1.patch

```diff
@@ -22,7 +22,7 @@ index 5ae42ec..da59568 100644
  export const getAnimatedStyle = (received) => {
      return getCurrentStyle(received);
 diff --git a/node_modules/react-native-reanimated/src/createAnimatedComponent.tsx b/node_modules/react-native-reanimated/src/createAnimatedComponent.tsx
-index 1cf0c3f..a334889 100644
+index 1cf0c3f..a242e4b 100644
 --- a/node_modules/react-native-reanimated/src/createAnimatedComponent.tsx
 +++ b/node_modules/react-native-reanimated/src/createAnimatedComponent.tsx
```

Comment:
Any chance we need to submit this upstream?

Reply:
https://github.com/software-mansion/react-native-reanimated/pull/3126/files